### PR TITLE
chore(packages): add ESLint, vscode extensions, and github workflows

### PR DESCRIPTION
**Pull request by nlewis84**
## 🐛 Issue

Switching between repos was causing havoc due to differences in lint/prettier setups

## ✏️ Solution

- Bring `web-embeds` in line with lint/prettier setup on Apollos Admin
- Add some `vscode` extensions config as well to make context switching easier

## 🔬 To Test

1. Run `yarn` in the `web-embeds` folder
2. Run `yarn format` and see no changes
3. Run `yarn lint` and see no changes

## 📸 Screenshots

Screenshot 2024-01-24 at 9 56 56 AM

### CODE REVIEW

**Comment by nlewis84** [comment]
Can't tell if the vercel deployment is failing because this is a draft. It's ready for review aside from the deploy issue.

File: web-embeds/jsconfig.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "compilerOptions": {
-    "baseUrl": "."
-  }
+    "baseUrl": ".",
+  },
```
Comment (vinnyjth) [inline web-embeds/jsconfig.json:4]:
```suggestion
  }
```
I think this is why `vercel` is failing

File: web-embeds/jsconfig.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "compilerOptions": {
-    "baseUrl": "."
-  }
+    "baseUrl": ".",
```
Comment (vinnyjth) [inline web-embeds/jsconfig.json:3]:
```suggestion
    "baseUrl": "."
```

File: .github/workflows/lint.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+name: Lint Web Embeds
```
Comment (vinnyjth) [inline .github/workflows/lint.yml:1]: Amazing, thank you!

Comment (vinnyjth) [inline .github/workflows/lint.yml:1]: Amazing, thank you!

File: web-embeds/public/index.html [hunk 4]
```diff
@@ -1,19 +1,21 @@
-<!DOCTYPE html>
+<!doctype html>
```
Comment (vinnyjth) [inline web-embeds/public/index.html:1]: my heart breaths easy now

**Review by vinnyjth — CHANGES_REQUESTED** [review]
Love it! Let's fix the vercel deploys and then I'm all in

**Comment by nlewis84** [comment]
@vinnyjth The vercel deploy is passing now.

**Review by vinnyjth — APPROVED** [review]
🎉

**Pull request merged**